src/content/docs/tuhura-tech/style.mdx

```diff
@@ -0,0 +1,342 @@
+---
+title: Contribution Best practices
+description: Details on how to best structure and layout your contributions
+---
+
+
+import Checklist from '/src/components/tutorial/Checklist.astro';
+import Box from '/src/components/tutorial/Box.astro';
+import { Steps } from '@astrojs/starlight/components';
+import { FileTree } from '@astrojs/starlight/components';
+
+:::note[Markdown]
+For more general info on Markdown syntax, take a look at the [Markdown Guide](https://www.markdownguide.org/basic-syntax/#overview)
```

Comment:
Technically we allow a superset of markdown due to it being astro/starlight so we should link to that instead.